src/beaconDecoder.py

```diff
@@ -30,17 +30,13 @@ def run(self):
             else:
                 return None
         except:
-            print("Invalid data received!")
```

Comment:
Don't you want to say that you at least got something?

Reply:
Gnuradio was giving us the noise it was picking up and this was constantly printing, which wasn't helpful